connector/src/main/scala/com/datastax/spark/connector/util/Threads.scala

```diff
@@ -0,0 +1,19 @@
+package com.datastax.spark.connector.util
+
+import java.util.concurrent.{Executors, LinkedBlockingQueue, ThreadPoolExecutor, TimeUnit}
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder
+
+import scala.concurrent.ExecutionContext
+
+object Threads {
+
+  implicit val BlockingIOExecutionContext = {
+    val threadFactory = new ThreadFactoryBuilder()
+      .setDaemon(true)
+      .setNameFormat("spark-cassandra-connector-io" + "%d")
+      .build
```

Comment:
perhaps it would be good to add exception handler which logs failures

Reply:
done